frontend/src/components/SearchBar/SearchBar.css

```diff
@@ -0,0 +1,63 @@
+.findText {
+  color: white;
+  font-size: 2vw;

```

Comment:
Would it be possible to do this CSS without using `vw` and `vh`? I'm going to be officially blacklisting these units soon via the linter, and avoiding them now will save some back-tracking.